docs/sql-ref-syntax-qry-select-setops.md

```diff
@@ -19,4 +19,125 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+Set operators are used to combine the results of two queries into a single result. Spark SQL supports three types of set operators:
```

Comment:
`to combine the results of two queries into a single result.` -> `to combine two input relations into a single one.`?